packages/suite/src/components/suite/Settings/components/Row.tsx

```diff
@@ -11,7 +11,7 @@ const Content = styled.div`
 const Wrapper = styled.div`
     display: flex;
     flex-direction: column;
-    & + & {
```

Comment:
is this not possible anymore? I am asking because there are maybe 30 occurences of this selector in the codebase

Reply:
It is possible, but it stopped working on this place. All other places look good